server/server.js

```diff
@@ -0,0 +1,32 @@
+const server = require('express')();
+const db = require('../data/dbConfig')
+
+server.use(require('express').json());
+
+server.get('/', (req, res) => {
+    // Sanity Check
+    res.send(`Server Home directory GET is active.`);
+  });
+  
+  server.post('/users', (req, res) => {
+    const newUser = req.body;
+    req.body.name 
+      ? db.insert(req.body)
```

Comment:
I figured out the source of the error keeping this from successfully posting--this isn't a helper function you still need to reference the table in the database such as `db('user').insert`, otherwise it will throw the error you're getting